### PR TITLE
adding readme file

### PR DESCRIPTION
**Pull request by Ahmed-Ghanem**

### CODE REVIEW

**Comment by kareemalised** [comment]
## Review

Looks **greate**! 👍

File: readme.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# starter web repo
```
Comment (kareemalised) [inline readme.txt]: test

Comment (kareemalised) [inline readme.txt]: test review

Comment (Ahmed-Ghanem) [reply readme.txt]: reply test

**Pull request merged**